### PR TITLE
Fixes #329 Update repository if hook is deleted

### LINKED ISSUE #329: Removing webhook from a registered repository and registering again leads to being displayed twice
### This is a
- [x] Bug report
- [ ] Feature request

### Current behaviour

screenshot 196

### PR DESCRIPTION
**Pull request by imujjwal96**
## Description

* Check if a repository is registered to yaydoc after the hook is registered.
* If the repository is registered, then update with access token and info of the new user
## Related Issue

issue #329 
## How Has This Been Tested?

Tested locally
## Types of changes

- [x] Bug fix 
- [ ] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

**Comment by imujjwal96** [comment]
@pri22296  @sch00lb0y  Please review.

File: model/repository.js [hunk 1]
```diff
@@ -71,3 +71,28 @@ module.exports.getRepositoriesByOwner = function (owner, callback) {
...
+};
+
+/**
+ * Update the repository if hook is deleted
+ * @param name: `full_name` of a repository
+ * @param data: `token`, `id` and `login` of a user
+ * @param callback
+ */
+module.exports.updateRepository = function (name, data, callback) {
+  Repository.findOneAndUpdate({name: name}, {$set: {
```
Comment (sch00lb0y) [inline model/repository.js]: can you try upsert instead of doing two queries?

**Comment by imujjwal96** [comment]
@sch00lb0y updated. Thanks for the tip. Didn't know about upsert

**Comment by imujjwal96** [comment]
@sch00lb0y @pri22296   Since this pr has 2 approved review. I would request if any one of you can squash and merge this.

**Pull request merged**